peef/peef.go

```diff
@@ -91,3 +96,33 @@ var (
 		},
 	}
 )
+
+func getSymbolCurrentPrice(symbol string, key string) float32 {
+	var prices []Price
+
+	url := fmt.Sprintf("https://financialmodelingprep.com/api/v3/quote-short/%s?apikey=%s", symbol, key)
```

Comment:
What do you think of building the query string instead of hardcoding it?

[StackOverflow Reference](https://stackoverflow.com/questions/30652577/go-doing-a-get-request-and-building-the-querystring)

Reply:
Definitely more elegant. Looks pretty easy too. I’ll give it a go.